src/main/resources/db/migration/V2023.06.19.0001__add_goal_related_tables.sql

```diff
@@ -0,0 +1,72 @@
+--- Table action_plan
+CREATE TABLE action_plan
+(
+    id            VARCHAR(36)  PRIMARY KEY,
+    prison_number VARCHAR(10) NOT NULL,
```

Comment:
Very nit-picky and minor - the alignment isn't quite right here! 😱

Reply:
🙈